patches/minecraft/net/minecraft/client/renderer/model/ModelManager.java.patch

```diff
@@ -5,7 +5,7 @@
  
     public void func_195410_a(IResourceManager p_195410_1_) {
 -      ModelBakery modelbakery = new ModelBakery(p_195410_1_, this.field_174956_b);
-+      net.minecraftforge.client.model.ModelLoader modelbakery = new net.minecraftforge.client.model.ModelLoader(p_195410_1_, this.field_174956_b);
++      ModelBakery modelbakery = new ModelBakery(p_195410_1_, field_174956_b);
```

Comment:
Could this patch not be dropped completely if `this` was added back?

Reply:
@Chocohead yes, good catch